### PR TITLE
Update docstrings, doctests, tutorial, & implementations to new interface

### PR DESCRIPTION
**Pull request by papajohn**
TODO
 - [ ] Resolve some warnings/errors when running `make docs`

### CODE REVIEW

**Comment by papajohn** [comment]
I still need to fix some errors with the docs generation, but wanted to give you a preview of a change I'd like to push soon. I believe it makes the docs & such coherent with the changes checked in last Friday for v0.5.0.

File: docs/tutorial.rst [hunk 1]
```diff
@@ -55,27 +55,27 @@ Creating a Table
...
+columns.
 
 .. ipython:: python
 
-    letters = ['a', 'b', 'c', 'z']
-    counts = [9, 3, 3, 1]
-    points = [1, 2, 2, 10]
+    letters =
+    counts =
+    points =
```
Comment (SamLau95) [inline docs/tutorial.rst]: Did you mean to delete these lines?

Comment (papajohn) [inline docs/tutorial.rst]: Fixed.

**Comment by papajohn** [comment]
Ok, I've fixed the tutorial. I flipped through some of the generated HTML docs, and they looked fine too me. Anything else we need to clean up to keep the code base and docs consistent? I didn't update the tests to the new interface because I'd like to make sure we still support the deprecated interface as well for a while, so that connector instructors don't get too surprised.

Also, @SamLau95 can you redeploy the docs after we merge, because I don't yet know how to do that? Thanks!

**Comment by SamLau95** [comment]
Looks good to me!

The docs should auto-push to the `master` branch of RTD after a merge: http://datascience.readthedocs.org/en/master/

I'll double check after the merge goes through.

**Comment by papajohn** [comment]
Great. Want to deploy to {data8,ds8}.berkeley.edu as well? Thanks.

**Comment by SamLau95** [comment]
Sure thing.

**Comment by ryanlovett** [comment]
Updated to 0.5.1 on both.

**Comment by SamLau95** [comment]
Beat me to it. Thanks Ryan.
ᐧ

On Sun, Jan 31, 2016 at 11:08 PM, Ryan Lovett <notifications@github.com>
wrote:

> Updated to 0.5.1 on both.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/data-8/datascience/pull/189#issuecomment-177817207>.
>



-- 
Sam Lau
EECS | UC Berkeley | May 2017
(562) 565-4336 | samlau95@berkeley.edu

**Comment by ryanlovett** [comment]
No problem.

**Pull request merged**